bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -31,6 +34,18 @@ class Edge extends WebBrowser {
 		Library.loadLibrary("WebView2Loader");
 	}
 
+	static {
+		URI absolutePath;
+		try {
+			Path tempFile = Files.createTempFile("base", ".html");
+			absolutePath = tempFile.toUri();
+			tempFile.toFile().deleteOnExit();
+		} catch (IOException e) {
+			absolutePath = URI.create("about:blank");
+		}
+		URI_FOR_CUSTOM_TEXT_PAGE = absolutePath;
```

Comment:
Just saw that there is already a static initializer block, so it would be good to integrate this into the other (though keeping the two functionalities separated in separate methods called by that initializer).

Reply:
Done